README.md

```diff
@@ -32,10 +30,8 @@ After you're finished please remove all the comments and instructions!
 
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>Ayomid s.e.p README </b></h3>
```

Comment:
- [ ] The title of your README should be according to the current project e.g. `Paulinelali Portfolio`, please 🙏🏽 fix it for professional documentation.